client/state/data-layer/wpcom/sites/comments/index.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * External dependencies
+ */
+import { translate } from 'i18n-calypso';
+import { forEach, groupBy } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { COMMENTS_LIST_REQUEST, COMMENTS_RECEIVE } from 'state/action-types';
+import { http } from 'state/data-layer/wpcom-http/actions';
+import { dispatchRequest } from 'state/data-layer/wpcom-http/utils';
+
+import { errorNotice } from 'state/notices/actions';
+
+// @see https://developer.wordpress.com/docs/api/1.1/get/sites/%24site/comments/
+const fetchCommentsList = ( { dispatch }, action ) => {
+	const { query } = action;
+	const { listType } = query;
+
+	if ( 'site' === listType ) {
+		const {
+			siteId,
+			status = 'unapproved',
+			type = 'comment',
```

Comment:
Not sure what we have in mind about it, but WP Admin (at least .org) gives the choice between `comments` and `pings` (both trackbacks and pingbacks, according to the api docs).
Now, on my site with definitely some really old pings, filtering doesn't show any results so maybe it's borked?